docs/code/SchemaComprehension.md

```diff
@@ -70,23 +71,23 @@ static void Main(string[] args)
     // Create the data view.
     // This method will use the definition of IrisData to understand what columns there are in the
     // data view.
-    var dv = context.Data.ReadFromEnumerable(dataArray);
+    var dataView = context.Data.LoadFromEnumerable(dataArray);
 
     // Now let's do something to the data view. For example, concatenate all four non-label columns
     // into 'Features' column.
     var pipeline = context.Transforms.Concatenate("Features",
         "SepalLength", "SepalWidth", "PetalLength", "PetalWidth");
 
     // Next, let's fit and transform the data so the concatenation goes through the data view.
-    var transformedData = pipeline.Fit(dv).Transform(dv);
+    var transformedData = pipeline.Fit(dataView).Transform(dataView);
 
     // Read the data into an IEnumerable.
     // This method will use the definition of IrisData to understand which columns and of which types
     // are expected to be present in the input data.
-    var data = context.CreateEnumerable<IrisData>(transformedData, reuseRowObject: false);
+    var data = context.Data.CreateEnumerable<IrisVectorData>(transformedData, reuseRowObject: false).ToList();
```

Comment:
This needs to be IrisData see line 64.

Reply:
If this must be `IrisData` then the next comment (line 89-90) makes no sense, since it won't contain the `Features` field.

Reply:
I'm sorry what you said is correct, I didn't realize this the transformed data and IrisVectorData wasn't defined anywhere.